FPS cap for IA/VT Colorful, fix PJDf/F2nd FPS patch random input spam

counter185 opened the pull request:
first PR in 2 years let's go

counter185 committed:
Add PCSG00355 framerate patch

counter185 committed:
[PCSG00355] keep style consistent

counter185 commented:
A few things to note about running this game at 60 FPS:
1. Most menu animations are made for 30 FPS, 60 is really only noticable in actual gameplay and while scrolling the song list.
2. Non-movie 3D PVs (the ones marked with `COSTUMES`) have small jittering issues (I'm looking into fixing this)
3. If you scroll the costumes menu too fast it may very rarely cause a GPU driver crash, anyone seriously playing this should keep autosave enabled

As for how this is possible, all this patch does is inject a custom FPS value into the `boot_config.ini` loading subroutine

yzzyx-network commented:
My favorite game on the Vita is now even better! Played with this patch enabled for roughly 4 hours and experienced no issues aside from the aforementioned jittering in the 3DPVs. Game held a steady 60fps with no noticeable drops during gameplay with overclocking turned on. Awesome job on this patch!

counter185 committed:
Fix random inputs in PJDF/PJDF2nd, add support for MMFE mod

counter185 commented:
It's been a while, I added a small fix to the FPS patch for Project Diva f and f 2nd, which used to spam random inputs by itself when the framerate dropped. I also added support for the Project Diva Megamix -F edition- total conversion mod (unfortunately the configuration app will not add its title ID to config.txt, it has to be done manually)

3cho0 commented:
Can you also add this fix for JP f and f2 (both have updates 1.01)? Unfortunately, there are a lot of edits only available on the JP version of the game.
